fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/rdf/RdfStream.java

```diff
@@ -13,12 +13,23 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+package org.fcrepo.kernel.api.rdf;
```

Comment:
OK, I will move RdfStream, RdfContext and RdfCollectors into the org.fcrepo.kernel.api package, but leave DefaultRdfStream in api.rdf for now. We should have a larger conversation about package naming and organization.